SQL (#1679)
enkidevs/curriculum: sql/dql/database-clients/node-database-packages.md

---
## Content

> Node is another popular platform (available as a course for premium users) which uses JavaScript. As there are lots of packages available for it we will take a look at some of the database packages:

Reviewer: The premium-ness should be dynamically explained and we can remove it from static content:

```suggestion
Node is another popular platform and it uses JavaScript. As there are lots of packages available for it we will take a look at some of the database packages:
```
